deno/lib/__tests__/discriminatedUnions.test.ts

```diff
@@ -0,0 +1,145 @@
+import { expect } from "https://deno.land/x/expect@v0.2.6/mod.ts";
+const test = Deno.test;
+
+import * as z from "../index.ts";
+
+describe("discriminated union", () => {
```

Comment:
Looking at the other deno tests, none of them are wrapped in a `describe` block, removing it here should fix the deno tests.

Reply:
Yeah, that was one thing... But I also had to get rid of the other jest functions. I simplified the tests so that they use only the test wrappers and the `expect(...).toEqual(...)` construct.